syntax = "proto3";

package rules;

import "rules/rule_result.proto";

service ResultService {
    rpc SendResult(RuleResult) returns (SendResultResponse);
}

message SendResultResponse {
    bool success = 1;
    string message = 2;
}
